pyxem/signals/beam_shift.py

```diff
@@ -18,17 +18,19 @@
 
 
 import numpy as np
-import pyxem.utils._pixelated_stem_tools as pst
+from scipy.ndimage import rotate
+import pyxem.utils._beam_shift_tools as bst
 from hyperspy._signals.lazy import LazySignal
-from hyperspy._signals.signal1d import Signal1D
+from hyperspy.signals import Signal1D
+from hyperspy.utils.plot import plot_images
 
 
 class BeamShift(Signal1D):
     """Signal class for working with shift of the direct beam."""
 
     _signal_type = "beam_shift"
 
-    def make_linear_plane(self, mask=None):
+    def get_linear_plane(self, mask=None, fit_corners=None):
```

Comment:
It might be worth testing this with a 5D dataset to make sure that it works with N-Dimensions.

Reply:
If we can assume that the probe positions are always the two first navigation dimensions, we could easily iterate over the remaining navigation dimensions.

We also need to handle `BeamShift` signals with 1 navigation dimension.